packages/auto-router/src/index.ts

```diff
@@ -1 +1,3 @@
+export * from './lib/savings-vault-router';
 export * from './lib/auto-router';
+export * from './lib/meta-router';
```

Comment:
I really want us to split this into 3 separate pkgs to optimize build and test time